src/wp-admin/includes/class-plugin-installer-skin.php

```diff
@@ -208,7 +208,7 @@ private function do_overwrite() {
 			return false;
 		}
 
-		echo '<h2 class="update-from-upload-heading">' . esc_html( __( 'This plugin is already installed.' ) ) . '</h2>';
+		echo '<h2 class="update-from-upload-heading">' . esc_html__( 'This plugin is already installed.' ) . '</h2>';
```

Comment:
Doesn't seem like the escaping is required at all here. The same applies to lines 224 and 225 which you have kept unchanged.

Reply:
Actually, I did change lines 224 and 225 but in #1189, which [landed](https://core.trac.wordpress.org/changeset/50705) in core.  I forgot to rebase, sorry.

Should we keep escaping for now and then remove it in all places, or should I remove it from this PR?